source/api_documentation/network_integration/advertiser_campaigns/index.rst

```diff
@@ -210,6 +210,22 @@ You are not allowed to delete campaigns.
     - integer
     - Maximum Promo Numbers.
 
+  * - **concurrent_call_cap_alert**
+    -
+    -
+
+  * - starts_at
+    - date
+    - Call Cap Start. Default: Beginning of day with the campaign's time zone.
+
```

Comment:
This is a little confusing for concurrent call caps, since they are continuous and don't really start at any particular time of day. Is this because we are following the pattern of other caps or can this be made more relevant to CCC?